assets/app/scripts/controllers/overview.js

```diff
@@ -289,26 +291,26 @@ angular.module('openshiftConsole')
         return;
       }
       // Make sure we have a build output
-      if (!build.parameters.output.to) {
+      if (!build.spec.output.to) {
         return;
       }
-      for (var i = 0; i < deploymentConfig.triggers.length; i++) {
-        var trigger = deploymentConfig.triggers[i];
+      for (var i = 0; i < deploymentConfig.spec.triggers.length; i++) {
+        var trigger = deploymentConfig.spec.triggers[i];
         if (trigger.type === "ImageChange") {
           var triggerImage = trigger.imageChangeParams.from.name;
-          var buildImage = build.parameters.output.to.name;
+          var buildImage = build.spec.output.to.name;
           if (triggerImage !== buildImage) {
             continue;
           }
 
           var triggerNamespace = trigger.imageChangeParams.from.namespace || deploymentConfig.metadata.namespace;
-          var buildNamespace = build.parameters.output.to.namespace || build.metadata.namespace;
+          var buildNamespace = build.spec.output.to.namespace || build.metadata.namespace;
           if (triggerNamespace !== buildNamespace) {
             continue;
           }
 
           var triggerTag = trigger.imageChangeParams.tag;
```

Comment:
`imageChangeParams.tag` no longer exists, the tag is baked into `imageChangeParams.from.name` when appropriate (`kind === "ImageStreamTag"`, for example)